imgparse/imgparse.py

```diff
@@ -646,8 +640,13 @@ def get_wavelength_data(image_path, exif_data=None, xmp_data=None):
     try:
         make, model = get_make_and_model(image_path, exif_data)
         xmp_tags = xmp.get_tags(make)
-        central_wavelength = parse_seq(xmp_data[xmp_tags.WAVELENGTH_CENTRAL], int)
-        wavelength_fwhm = parse_seq(xmp_data[xmp_tags.WAVELENGTH_FWHM], int)
+        try:
```

Comment:
Should I remove the try/ excepts here and put them in parse_seq?

Reply:
I'll leave it up to you. It would be cleaner to add to `parse_seq` directly, but it's working fine as is